JWBestPracticeApps/Recommendations/Recommendations/ViewController.swift

```diff
@@ -38,19 +41,30 @@ class ViewController: JWPlayerViewController {
         do {
             // First, use the JWPlayerItemBuilder to create a JWPlayerItem that will be used by the player configuration.
             // Add a URL to the recommendations feed. This will cause the recommendations menu to become available.
-            let playerItem = try JWPlayerItemBuilder()
+            let playerItemBuilder = JWPlayerItemBuilder()
                 .file(videoUrl)
                 .posterImage(posterUrl)
-                .recommendations(recommendationsUrl)
-                .build()
+
+            // Configure recommendations per player item
+            if useRecPerPlayerItem {
```

Comment:
To make this more clear, why not combine this if statement with the one on line 62? This way the logic is in one place for where to put the recommendations url.